src/Microsoft.Health.Fhir.Web/appsettings.json

```diff
@@ -31,6 +31,9 @@
             "Headers": [],
             "MaxAge": null,
             "AllowCredentials": false
+        },
+        "Operations": {
+            "SupportsExport": false
```

Comment:
Is there any type of configuration that we're going to need beyond if export is supported or not?

Reply:
I don't think there will be more configurations under export. We will be adding more to Operations as we support more (example: import).

Reply:
Seems like I am mistaken. There are other settings we want to add when export is enabled (like concurrency, job timeout, maximum filesize. It seems like a good idea to move the export config into a separate section instead of grouping under "Operations"

Reply:
We can either move it to it's own section or could just make each thing under operations an object. Not sure right now which would be the better decision.

Reply:
I just finished renaming everything to be Export specific (rather than Operations) so that things look cleaner and more readable (at least for now). If we add more operations, I thought it would be better to have their own controllers and appropriate filters.